.gitignore

```diff
@@ -15,8 +15,3 @@ sway-lib-std/Forc.lock
 
 # Forc's build directory
 out
-
-# Don't lock example or lib-std test harnesses in order to try and catch
-# accidental upstream patch release breakage before users do.
```

Comment:
This reasoning is a bit unsustainable, and could apply to any testing. It's impractical to allow unrelated bugs caused by spurious breaking patch releases to impact our regular workflow.

We can always be sure to catch these issues before users as long as we remember to run `cargo update` (on both the root and sdk-harness workspaces) as a part of our pre-release PR process.

Reply:
Maybe we should have a proper release script or ci action to make sure that we always run the same stuff?

Reply:
Yeah that sounds like a great idea :heavy_plus_sign: